getting_started/scripting/gdscript/gdscript_exports.rst

```diff
@@ -37,99 +39,102 @@ Examples
     # If the exported value assigns a constant or constant expression,
     # the type will be inferred and used in the editor.
 
-    export var number = 5
+    @export var number = 5
 
-    # Export can take a basic data type as an argument, which will be
-    # used in the editor.
+    # If there's no default value, you can add a type to the variable.
 
-    export(int) var number
+    @export var number: int
 
-    # Export can also take a resource type to use as a hint.
+    # Export works with resource types.
 
-    export(Texture) var character_face
-    export(PackedScene) var scene_file
+    @export var character_face: Texture
+    @export var scene_file: PackedScene
     # There are many resource types that can be used this way, try e.g.
     # the following to list them:
-    export(Resource) var resource
+    @export var resource: Resource
 
     # Integers and strings hint enumerated values.
 
     # Editor will enumerate as 0, 1 and 2.
-    export(int, "Warrior", "Magician", "Thief") var character_class
-    # Editor will enumerate with string names.
-    export(String, "Rebecca", "Mary", "Leah") var character_name
+    @export_enum("Warrior", "Magician", "Thief") var character_class
+    # If type is String, editor will enumerate with string names.
+    @export_enum("Rebecca", "Mary", "Leah") var character_name: String
 
     # Named enum values
 
     # Editor will enumerate as THING_1, THING_2, ANOTHER_THING.
     enum NamedEnum {THING_1, THING_2, ANOTHER_THING = -1}
-    export(NamedEnum) var x
+    @export var x: NamedEnum
```

Comment:
This doesn't seem to work for me in the current `master` branch (godotengine/godot@a6bb4f70e07edc52cb969548876a0b57abeef546).

Reply:
Yeah, I need to fix the export because it's missing some stuff still.